debian/control

```diff
@@ -86,6 +86,7 @@ Build-Depends: cmake (>= 3.10.2),
                python3-sphinx,
 # Make-Check   python3-werkzeug,
                python3-setuptools,
+               python3-routes,
```

Comment:
python3-cherrypy3 depends on python3-routes. see https://packages.ubuntu.com/eoan/python3-cherrypy3

so i don't think this is necessary.

Reply:
Well, I try to remove python3-cherrypy3 and python3-routes packages, and then run the command "./install-deps". Actually, it just install python3-cherrypy3 package and won't install python3-routes package. 

The problem may be that python3-routes is a recommendation package rather than a dependency package.

Reply:
right, it's not a hard dependency of `python3-cherrypy3`.

> When developers install ceph at PC(Ubuntu),

what do you mean by "install ceph"? please note, https://github.com/ceph/ceph/blob/7ffa753be032a3c6da58fc268a56c79c090b4b9b/debian/control#L249 so if you install "ceph-mgr-dashboard" package, `python3-routes` should be installed.

> the dashboard will not be installed correctly.

could you expand a little bit on this ?

but even if it is needed for testing, we should not add it as a `Build-Depends`.

Reply:
In short, I follow the documentation

`
./install-deps && 
./do_ cmake.sh && 
cd build &&
make
`

After these work done, I found that dashboard didn't work correctly. So I think that the addition of this package can make it easier for developers to get started.

Reply:
@ghyer i am not sure what "dashboard didn't work correctly" means. so i am conservative on adding dependencies for this use case. as in theory, `debian/control` is for preparing the build-time and runtime dependencies. we also use for tracking the "make check" dependencies as well, but i don't think it's for preparing a full-blown testing environment for a developer.

i am adding dashboard as a reviewer to see what they think.

Reply:
@tchaikov 
OK, thanks. 
"didn't work correctly" means:

> ceph -s
cluster:
    id: 8754e3c7-b632-4438-8c33-a15f06959639
    health: HEALTH_ERR
    Module 'dashboard' has failed: No module named routes

Reply:
@ghyer could you paste the backtrace in `out/mgr.x.log` ? i think it will help us understand under what circumstances it fails.

Reply:
@tchaikov emm... I have fixed the problem already, and I'm working for another function of dashboard now. So it's not very convenient to reproduce this bug.
But I can talk about the reproduction process:

> git clone https://github.com/ceph/ceph
cd ceph
./install-deps.sh && ./do_ cmake.sh && cd build && make
../src/vstart -n
./bin/ceph -s
>
And then the problem comes up:
>cluster:
id: 8754e3c7-b632-4438-8c33-a15f06959639
health: HEALTH_ERR
Module 'dashboard' has failed: No module named routes
>

Reply:
@ghyer on which version of Ubuntu did you experience the issue?

Reply:
@Devp00l 18.04 and my kernel infomation :

> root@main:/home/hello# uname -a
Linux main 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux